install-elasticsearch.rst

```diff
@@ -167,11 +172,36 @@ After you have configured Zammad for using Elasticsearch, you need to rebuild th
  zammad run rake searchindex:rebuild
 
 
+Adjust default settings of elasticsearch
+========================================
+
+To ensure an optimal performance of Zammad together with elasticsearch, you might want to increase the maximum possible 
```

Comment:
A note that the following are only example settings and need to be adjusted by everybody it self would be good. Because the settings are related to the local environments.